### ISSUE TITLE
[Request] import item value overrides

### ISSUE THREAD

**Issue by App-App** [Alumian/CEPGP-Retail#241]
My guild has a different structure for assigning GP to loot - so I have to manually override each single item when a raid is released.

Is it it possible to have the add accept a csv to import to make this process much easier (and potentially export as well?)

Thank you!

**Comment by scarecr0w12** [comment]
You can already import Points via a CSV document.

1. Type in /cep show
2. Click "Guild" in top left.
3. Click "Import standings" 
4. Follow instructions on right side.

**Comment by Alumian** [comment]
> You can already import Points via a CSV document.
> 
> 1. Type in /cep show
> 2. Click "Guild" in top left.
> 3. Click "Import standings"
> 4. Follow instructions on right side.

Hey Scare, I believe they're asking about importing item overrides, not so much EPGP standings.

@App-App I can add this, but you would still need to supply the same information as what is already required to override.

**Comment by scarecr0w12** [comment]
> > You can already import Points via a CSV document.
> > 
> > 1. Type in /cep show
> > 2. Click "Guild" in top left.
> > 3. Click "Import standings"
> > 4. Follow instructions on right side.
> 
> Hey Scare, I believe they're asking about importing item overrides, not so much EPGP standings.
> 
> @App-App I can add this, but you would still need to supply the same information as what is already required to override.

Yup, realized that after i re-read it, removed my comment to avoid confusion.

**Comment by App-App** [comment]
I’m talking about when you do override values for GP (assigning new GP values to individual pieces of loot) - currently that is only doable one piece at a time by entering the name, the wowhead item number, and the new GP value

**Comment by App-App** [comment]
Thanks - and yea - I expect to need all the info - but just doing it all on an excel sheet and then converting it to csv to upload would save tons of time haha

Thanks again
